myriad-scheduler/src/main/java/org/apache/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java

```diff
@@ -218,7 +218,7 @@ private boolean checkAggregates(Offer offer, NodeTask taskToLaunch, int ports, d
     if (aggrCpu <= cpus && aggrMem <= mem && taskConstraints.portsCount() <= ports) {
       return true;
     } else {
-      LOGGER.info("Offer not sufficient for task with, cpu: {}, memory: {}, ports: {}", aggrCpu, aggrMem, ports);
+      LOGGER.error("Offer not sufficient for task with, cpu: {}, memory: {}, ports: {}", aggrCpu, aggrMem, ports);
```

Comment:
Cool, will set to debug

On Wed, Jul 13, 2016 at 8:46 AM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/main/java/org/apache/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java
> <https://github.com/apache/incubator-myriad/pull/84#discussion_r70617680>:
>
> > @@ -218,7 +218,7 @@ private boolean checkAggregates(Offer offer, NodeTask taskToLaunch, int ports, d
> >      if (aggrCpu <= cpus && aggrMem <= mem && taskConstraints.portsCount() <= ports) {
> >        return true;
> >      } else {
> > -      LOGGER.info("Offer not sufficient for task with, cpu: {}, memory: {}, ports: {}", aggrCpu, aggrMem, ports);
> > +      LOGGER.error("Offer not sufficient for task with, cpu: {}, memory: {}, ports: {}", aggrCpu, aggrMem, ports);
>
> This isn't an error condition. It's expected some mesos resources will be
> less than needed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/84/files/b4fbad0582bb71b35968b1cdfb4e85e629ec3758#r70617680>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKSRgRSSRiLe6lkHzxePvWldFIXGT8x6ks5qVN41gaJpZM4JLUAy>
> .
>